astropy/io/votable/tree.py

```diff
@@ -3523,6 +3523,45 @@ def _get_version_checks(self):
             util.version_compare(self.version, '1.4') >= 0
         return config
 
+    # Map VOTable version numbers to namespace URIs and schema information.
+    _version_namespace_map = {
+        # Version 1.0 isn't well-supported, but is allowed on parse (with a warning).
+        # It used DTD rather than schema, so this information would not be useful.
+        # By omitting 1.0 from this dict we can use the keys as the list of versions
+        # that are allowed in various other checks.
+        "1.1": {
+            "namespace_uri": "http://www.ivoa.net/xml/VOTable/v1.1",
+            "schema_location_attr": "xsi:noNamespaceSchemaLocation",
+            "schema_location_value": "http://www.ivoa.net/xml/VOTable/v1.1"
+        },
+        "1.2": {
+            "namespace_uri": "http://www.ivoa.net/xml/VOTable/v1.2",
+            "schema_location_attr": "xsi:noNamespaceSchemaLocation",
+            "schema_location_value": "http://www.ivoa.net/xml/VOTable/v1.2"
+        },
+        # With 1.3 we'll be more explicit with the schema location.
+        # - xsi:schemaLocation uses the namespace name along with the URL
+        #   to reference it.
+        # - For convenience, but somewhat confusingly, the namespace URIs
+        #   are also usable URLs for accessing an applicable schema.
+        #   However to avoid confusion, we'll use the explicit schema URL.
+        "1.3": {
+            "namespace_uri": "http://www.ivoa.net/xml/VOTable/v1.3",
+            "schema_location_attr": "xsi:schemaLocation",
+            "schema_location_value":
+            "http://www.ivoa.net/xml/VOTable/v1.3 http://www.ivoa.net/xml/VOTable/VOTable-1.3.xsd"
```

Comment:
If http://www.ivoa.net/xml/VOTable/v1.3 is the home of future VOTable versions as well, should this be set as another variable? Setting as variable only makes sense if there is a chance of this URL changing for all affected VOTable versions in the future. If that is not going to happen, then we don't have to.

Reply:
That will be the namespace URI for future versions 1.x.  I think there is no chance the value will change though.  

Although normally I might use a variable just to avoid duplicate copies of the same string, for this case I think having these declared in-line without references to a variable makes it more explicit.